app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -77,41 +100,25 @@ class ExoPlayerProfile(
 						CodecTypes.MPEG2VIDEO
 					).joinToString(",")
 
-					audioCodec = if (Utils.downMixAudio()) {
-						arrayOf(
-							CodecTypes.AAC,
-							CodecTypes.MP3,
-							CodecTypes.MP2
-						).joinToString(",")
-					} else {
-						listOfNotNull(
-							CodecTypes.AAC,
-							CodecTypes.AC3,
-							CodecTypes.EAC3,
-							CodecTypes.AAC_LATM,
-							CodecTypes.MP3,
-							CodecTypes.MP2,
-							CodecTypes.DCA,
-							CodecTypes.DTS,
-							CodecTypes.OPUS,
-						).joinToString(",")
-					}
+					audioCodec =
+						when {
+							Utils.downMixAudio() -> downmixSupportedAudioCodecs.joinToString(",")
+							else -> allSupportedAudioCodecs.joinToString(",")
+						}
 				})
 			}
 			// Audio direct play
-			add(audioDirectPlayProfile(
-				CodecTypes.AAC,
-				CodecTypes.MP3,
-				CodecTypes.MPA,
-				CodecTypes.FLAC,
-				CodecTypes.WAV,
-				CodecTypes.WMA,
-				CodecTypes.MP2,
-				ContainerTypes.OGG,
-				ContainerTypes.OGA,
-				ContainerTypes.WEBMA,
-				CodecTypes.APE,
-				CodecTypes.OPUS
+			add(audioDirectPlayProfile(allSupportedAudioCodecs + arrayOf(
+						CodecTypes.MPA,
+						CodecTypes.FLAC,
+						CodecTypes.WAV,
+						CodecTypes.WMA,
+						ContainerTypes.OGG,
+						ContainerTypes.OGA,
+						ContainerTypes.WEBMA,
+						CodecTypes.APE,
+						CodecTypes.OPUS,
+					)
```

Comment:
```suggestion
			add(audioDirectPlayProfile(allSupportedAudioCodecs + arrayOf(
					CodecTypes.MPA,
					CodecTypes.FLAC,
					CodecTypes.WAV,
					CodecTypes.WMA,
					ContainerTypes.OGG,
					ContainerTypes.OGA,
					ContainerTypes.WEBMA,
					CodecTypes.APE,
					CodecTypes.OPUS,
				)
```

Reply:
This built in formatter is fighting me on this and constantly shifts it back to:
```
			add(
				audioDirectPlayProfile(
					allSupportedAudioCodecs + arrayOf(
						CodecTypes.MPA,
						CodecTypes.FLAC,
						CodecTypes.WAV,
						CodecTypes.WMA,
						ContainerTypes.OGG,
						ContainerTypes.OGA,
						ContainerTypes.WEBMA,
						CodecTypes.APE,
						CodecTypes.OPUS,
					)
				)
			)
```

Do we want to hand-edit the formatting each time or go with the formatter in this case?

Reply:
I avoid auto-formatting for this reason. I just highlight lines and do tab/shift + tab to bulk +/- indentation

Reply:
The suggestions I made are a result of formatting in Android Studio, the only change in the settings I made is to use tabs.

Reply:
```
			add(audioDirectPlayProfile(allSupportedAudioCodecs + arrayOf(
				CodecTypes.MPA,
				CodecTypes.FLAC,
				CodecTypes.WAV,
				CodecTypes.WMA,
				ContainerTypes.OGG,
				ContainerTypes.OGA,
				ContainerTypes.WEBMA,
				CodecTypes.APE,
				CodecTypes.OPUS,
			)))
```

This snippet works fine for me, formatter doesn't change it

Reply:
Huh weird, I'm on 2021.1.1 so I'm not sure if the internal formatter has changed?
I'll do it by hand anyway :+1: